docs/chapters/02.txt

```diff
@@ -143,30 +150,34 @@ There are some points to note about this diagram.
    means pair.
 
 So what, precisely, is the "relation" here? Well, it's the whole
-thing. The relationship is the set of all the ordered pairs, and
+thing. The *relationship* is the set of all the ordered pairs, and
 the ordering itself (i.e., how part A relates to part B). What we
-have in the preceding diagram is a picture of a relation and nothing
+have in the preceding diagram is a picture of a *relation* and nothing
 but a relation.
 
 This relation is a set. It has members. The members define the
 set. However, the members are no longer "elements", but ordered
 pairs. There's no reason to limit ourselves to ordered pairs,
-though. That's just the simplest relation, the binary relation,
+though. That's just the simplest relation, the *binary relation*,
 which is sometimes called "a relation of degree 2" (because there
 are two columns). We could have relations of degree 3, degree 4,
 degree 5, degree 6, degree 7, and so on, i.e., relations which are not
 binary but triple, quadruple, pentuple, sextuple, septuple ...
 Did you notice how after a while all the words ended in
-"-tuple"? That's why the general term for a relation with n
-elements is n-tuple. Here is a relation of degree 4 showing all
+"-tuple"? That's why the general term for a relation with *n*
+elements is *n-tuple*. Here is a relation of degree 4 showing all
 the information we have so far:
 
-::
++----------------+--------------+--------------------+-------------------+
+| ``NAME``       | ``ACTIVITY`` | ``RESIDENCE``      | ``DATE_OF_DEATH`` |
++================+==============+====================+===================+
+| Georg Cantor   | Set Theory   | Mental Institution | 1918              |
++----------------+--------------+--------------------+-------------------+
+| Kaiser Wilhelm | World War    | I Imperial Palace  | ???               |
++----------------+--------------+--------------------+-------------------+
+| ...            | ...          | ...                | ...               |
```

Comment:
looking at the book I now think that "..." should just be on a line by itself under the table

I don't know what it's there for, but the book has it, so... :)